docs/style-guide.rst

```diff
@@ -112,6 +112,136 @@ No::
         }
     }
 
+Maximum Line Length
+===================
+
+Keeping lines under the `PEP 8 recommendation <https://www.python.org/dev/peps/pep-0008/#maximum-line-length>`_ of 79 (or 99) 
+characters helps readers easily parse the code.
+
+Wrapped lines should conform to the following guidelines.
+
+1. The first argument should not be attached to the opening parenthesis. 
+
+2. One, and only one, indent should be used.
+
+3. Each argument should fall on its own line.
+
+4. The closing parenthesis and semi-colon is placed on the final line.
+
+Function Calls
+
+Yes::
+
+    thisFunctionCallIsReallyLong.(
+        longArgument1, 
+        longArgument2, 
+        longArgument3);
+
+No::
+
+    thisFunctionCallIsReallyLong.(longArgument1, 
+                                  longArgument2, 
+                                  longArgument3);
+                                  
+    thisFunctionCallIsReallyLong.(longArgument1, 
+        longArgument2, 
+        longArgument3);                                  
+                                  
+    thisFunctionCallIsReallyLong.(
+        longArgument1, longArgument2,
+        longArgument3);                                    
+
+    thisFunctionCallIsReallyLong.(
+    longArgument1, 
+    longArgument2, 
+    longArgument3);
+
+    thisFunctionCallIsReallyLong.(
+        longArgument1, 
+        longArgument2, 
+        longArgument3
+    );        
+
+Assignment Statements
+
+Yes::
+
+    thisIsALongNestedMapping[being][set][to_some_value] = someFunction(
+        argument1,
+        argument2,
+        argument3,
+        argument4);
+
+No::
+
+    thisIsALongNestedMapping[being][set][to_some_value] = someFunction(argument1,
+                                                                       argument2,
+                                                                       argument3,
+                                                                       argument4);
+
+Event Definitions and Event Emitters
+
+Yes::
+
+    event LongAndLotsOfArgs(
+        adress sender,
+        adress recipient,
+        uint256 publicKey,
+        uint256 amount,
+        bytes32[] options);
+
+    LongAndLotsOfArgs(
+        sender,
+        recipient,
+        publicKey,
+        amount,
+        options);
+
+No::
+
+    event LongAndLotsOfArgs(adress sender,
+                            adress recipient,
+                            uint256 publicKey,
+                            uint256 amount,
+                            bytes32[] options);
+
+    LongAndLotsOfArgs(sender,
+                      recipient,
+                      publicKey,
+                      amount,
+                      options); 
+
+Output Parameters and Return Statements
+
+Yes::
```

Comment:
When possible, return statements should syntactically correspond to their respective output parameters